packages/functional-tests/tests/signUp/signUpWithCode.spec.ts

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { test, expect } from '../../lib/fixtures/standard';
+
+const PASSWORD = 'passwordzxcv';
+let email;
+
+test.describe('Sign up with code ', () => {
+  test.beforeEach(async ({ pages: { login } }) => {
+    test.slow();
+    email = login.createEmail();
+    await login.clearCache();
+  });
+
+  test('bounced email', async ({ target, page, pages: { login } }) => {
+    const client = await login.getFxaClient(target);
+    await page.goto(`${target.contentServerUrl}`, {
```

Comment:
tiny nit: Since there aren't any other variables in the template string, we could probably just use the `target.contentServerUrl` value directly and save a fraction of a millisecond. But we could just as easily ignore this comment.
```suggestion
    await page.goto(target.contentServerUrl, {
```